### ISSUE TITLE
[🐞]: Server config does not respect "rope_override" setting for rope arrows

### ISSUE THREAD

**Issue by sapphiriza** [MehVahdJukaar/Supplementaries#1063]
### Before Continuing:

- [X] Check you are using the latest version of the mods and its dependencies
- [X] Always include a latest.log if you are crashing
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version

1.20.1-FABRIC

### Supplementaries Version

1.20-2.8.10

### Moonlight Lib Version

1.20-2.11.33

### Issue Detail

Changing "rope_override": "farmersdelight:rope" on client & server, rope arrows will still place Supplementaries rope when fired.

Using Quilt if that's relevant.

**Comment by MehVahdJukaar** [comment]
so i cant replicate.  Btw you only need to change it on server config, its the one that decides it

**Comment by sapphiriza** [comment]
Unfortunately I'm testing and I do consistently see this issue still after re-verifying, with minimal mods installed. This is the config I have on the server. When I shoot a rope arrow on the server, the Supplementaries Rope is placed, when the same config file on the client makes the rope arrow
[supplementaries-common.json](https://github.com/user-attachments/files/15812423/supplementaries-common.json)
The config file is located in the config/ folder, supplementaries-common.json
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/120682601/393a7f81-a12f-4b6e-b5ad-ecdc6e9286e5)

**Mod Loader & Versions:**
Quilt Loader: 0.26.0
Farmer's Delight: 1.20.1-2.1.1+refabricated
Moonlight: 1.20-2.11.33
QFAPI: 7.5.0+0.92.2-1.20.1
Supplementaries: 1.20-2.8.15

Let me know if there's any other information I can give to see why you cannot replicate.

**Comment by MehVahdJukaar** [comment]
does it show in the tooltip? does it work insingle player?

**Comment by sapphiriza** [comment]
Yes, the same config applied to the instance DOES work for single player: changing the placed rope from rope arrows to be the farmer's delight rope.

Good catch! When on the Server, the tooltip DOES show that it's configured to be farmer's delight rope, but it still places supplementaries:rope. And it doesn't matter what the client is configured as, the extra tooltip line doesn't show up in Singleplayer if the config isn't set. So the server does load the proper config (even when client config doesn't match), but the placement logic doesn't respect that setting on the server.
![image](https://github.com/MehVahdJukaar/Supplementaries/assets/120682601/61699726-2a56-4640-8d8a-73daac42ecd1)
